lib/author.rb

```diff
@@ -0,0 +1,28 @@
+class Author
+  attr_reader :first_name, :last_name, :books
+
+    def initialize(author_names)
+      @first_name = author_names[:first_name]
+      @last_name = author_names[:last_name]
+      @books = []
+    end
+
+    def name
+      @first_name + " " + @last_name
+
+    end
+
+    def write(title, date)
+      book = Book.new({:author_first_name => @first_name, :author_last_name => @last_name, :title => title, :publication_date => date})
+      @books << book
+      book
+
+
+    end
+
+
+
+
+
+
+end
```

Comment:
good job in here; try to clean up extra whitespace though once you're done building/testing/debugging